libs/components/src/lib/text-field/text-field.scss

```
@@ -81,12 +82,8 @@ $low-ink-color: --_low-ink-color;
 	}
 
 	/* Size */
-	.base:not(.density-extended)>& {
-		block-size: 40px;
-	}
-
-	.base.density-extended>& {
-		block-size: 48px;
+	.base > & {
+		block-size: #{size.$vvd-size-normal};
```

```suggestion
	.base:not(.density-extended)>& {
		block-size: #{size.$vvd-size-normal};
	}

	.base.density-extended>& {
		block-size: #{size.$vvd-size-expanded};
```
we have [this ](https://github.com/Vonage/vivid-3/issues/718) and [this](https://github.com/Vonage/vivid-3/issues/641#:~:text=change%C2%A0%23718-,%5Btext%2Dnumber%5D%3A%20remove%20density%20%2B%20add%20use%2Dcase%20with%20token%20value%20change%C2%A0%23743,-%5Bsize%5D%3A%20change%20all) for omitting the size in text-area + number-field.